public/src/main/java/com/rackspace/salus/telemetry/api/web/TestMonitorController.java

```diff
@@ -53,4 +62,14 @@ public TestMonitorController(ServicesProperties servicesProperties) {
     return proxy.uri(backendUri)
         .post();
   }
+
+  @ResponseStatus(HttpStatus.OK)
+  @PostMapping("/tenant/{tenantId}/test-monitor-event-task")
+  public TestMonitorAndEventTaskResponse createTestMonitorAndEventTask(
```

Comment:
I missed this before, please rename:
```suggestion
  public TestMonitorAndEventTaskResponse performTestMonitorAndEventTask(
```

Reply:
Renamed.